src/ol/renderer/canvas/VectorImageLayer.js

```diff
@@ -105,6 +105,16 @@ class CanvasVectorImageLayerRenderer extends CanvasImageLayerRenderer {
     return !!this.image_;
   }
 
+  /**
+   * @override
+   */
+  preRender() {}
+
+  /**
+   * @override
+   */
+  postRender() {}
```

Comment:
Not reading on a full sized computer, but won’t the vector image layer need to implement these to get events?

Reply:
The vector image layer renderer relies on the vector layer renderer, which dispatches these events already. Without this noop override, the events would be triggered twice.